inc/saiport.h

```diff
@@ -1955,6 +1955,19 @@ typedef enum _sai_port_attr_t
      */
     SAI_PORT_ATTR_DUAL_MEDIA,
 
+    /**
+     * @brief IPG
```

Comment:
maybe some couple words in brief information? not just 3 letters?

Reply:
added brief